contracts/core/BaseAccount.sol

```diff
@@ -17,19 +18,9 @@ abstract contract BaseAccount is IAccount {
     using UserOperationLib for UserOperation;
 
     //return value in case of signature failure, with no time-range.
-    // equivalent to packSigTimeRange(true,0,0);
+    // equivalent to _packSigTimeRange(true,0,0);
```

Comment:
Maybe it's time to rename it to something like packedValidationData and add doc: 
/**
 * packedValidationData is a packed representation of the data used for validation.
 * It is packed with the following fields:
 * - signature check status or aggregator address (uint160)
 * - validFrom (uint48)
 * - validUntil (uint48)
 */